### DISCUSSION TITLE
eForms SDK 1.2.0 is available

### DISCUSSION THREAD

**Discussion by bertrand-lorentz** [OP-TED/eForms-SDK#143] [Announcements]
We have released version 1.2.0 of the [eForms Software Development Kit](https://github.com/OP-TED/eForms-SDK/releases/tag/1.2.0).

This release brings various changes in the metadata content.

Please see the [release notes](https://github.com/OP-TED/eForms-SDK/releases/tag/1.2.0) for more details.

The release artifact is available for [download in Maven Central](https://search.maven.org/search?q=g:%22eu.europa.ted.eforms%22%20AND%20a:%22eforms-sdk%22). It might take a few hours for the new version to appear in searches.

You can read more on what this new major release means, and what is coming next, [in the eForms SDK roadmap](https://docs.ted.europa.eu/home/eforms/roadmap/index.html).

**Comment by IS-gmitov** [comment]
Hi, 

I see that from yesterday (10.10.2022) eForms SDK 1.2.1 is available. Why the section Metadata Reference on [https://docs.ted.europa.eu/eforms/1.2.1/index.html] is deleted? It was extremely useful. As I see the information about eForms SDK 1.2.0 is deleted, and the Section Metadata Reference is deleted from 1.1.1 too, so its impossible to see that information from the older versions. Can you put it back?

**Reply by bertrand-lorentz** [reply]
It seems something has gone wrong with the latest publication on the docs website. Those removals were not intentional.
We are looking into it, and will bring back the missing pages as soon as possible.

Sorry for the inconvenience !

**Reply by bertrand-lorentz** [reply]
The docs website should now be OK: the "Metadata Reference" section is now present for both 1.1.1 and 1.2.1.
https://docs.ted.europa.eu/eforms/1.1.1/reference/index.html
https://docs.ted.europa.eu/eforms/1.2.1/reference/index.html

**Comment by IS-gmitov** [comment]
It seems everything is fine now. Thank you!
